ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_search.py

```diff
@@ -295,3 +295,39 @@ def test_paddle_topk(
         sorted=sorted,
         test_values=False,
     )
+
+
+# where
+@handle_frontend_test(
+    fn_tree="paddle.where",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_dim_size=1,
+    ),
```

Comment:
This feels very simple for a `where` function test, if I were you I'd look at other `test_frontend_where` functions to see how they generate their test data since it's likely to be very similar for paddle